pages/your_tasks/real_dev_environments.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Fast, Declarative, Reproducible and Composable Developer Environments using Nix 
+description: "Going a step beyond just \"virtual envirnments\" - with advantages and drawbacks"
+contributors:
+  - "Nicole Dresselhaus"
+page_id: developer_environments
+related_pages:
+  your_tasks: 
+    - virtual_environments
+---
+
+## How are devenvs different from virtual software development environments?
+
+Developer Environments (short {% tool "devenv" %}s) are a more holistic
+approach to the classical virtual environments. Virtual environments - despite
+their undisputed advantages - tend to break at similar steps in the real world.
+An environment working on one machine might break on another because of
+different operating system, the existence or absence of globally installed
+packages, different hardware etc.
+You local software might behave different as a docker-container, yield different
+results or may not run at all.
+
+## Advantages and Drawbacks
+
+### Advantages
+
+- The same definition runs nearly everywhere the same.
+- Generating a docker-image of your software is included
+- Automatically setting up multiple services that get started (and even seeded)
+  automatically and can be used reproducibly, i.e.
```

Comment:
This sounds more like Ansible, or other tools for setup/deployment automation. I didn't see this mentioned in the intro, but perhaps it will be after a description of devenv itself is added? :)